bartimaeus_gui_main.py

```diff
@@ -76,7 +76,7 @@ def __init__(self):
         self.trash_button.grid(row = 6, column = 0, padx =20, pady =10)
         
         #side frame text
-        self.side_label = ck.CTkLabel(self.side_frame, text="Steve\'s \n voice-activated \n email sender",text_color='#fff',font=ck.CTkFont('stylus',30))
+        self.side_label = ck.CTkLabel(self.side_frame, text="Bartimeus \n a voice-activated \n email sender",text_color='#fff',font=ck.CTkFont('stylus',30))

```

Comment:
Spell check on **Bartimeus**: **Bartimaeus** not **Bartimeus**. I think we should also add a colon afterwards like this -> **"Bartimaeus:" \n a voice-activated \n email sender"**. Or **Bartimaeus** would be in a bigger font size while **"\n a voice-activated \n email sender"** would be in a smaller font and perhaps different font type.